src/SOS/SOS.UnitTests/SOSRunner.cs

```diff
@@ -562,6 +562,7 @@ public static async Task<SOSRunner> StartDebugger(TestInformation information, D
                         throw new ArgumentException("LLDB helper script path not set or does not exist: " + lldbHelperScript);
                     }
                     arguments.AppendFormat(@"--no-lldbinit -o ""settings set interpreter.prompt-on-quit false"" -o ""command script import {0}"" -o ""version""", lldbHelperScript);
+                    arguments.Append(@" -o ""settings set target.disable-aslr false""");
```

Comment:
@mikem8361, this was the actual fix for successfully running lldb tests in newer CentOS7 images. Took me a while to figure it out. 😅

There is a related discussion about lldb packet messaging with server at https://github.com/llvm/llvm-project/issues/48067, which helped figuring this one out i.e. why we need to set `disable-aslr` to `false` in addition to passing `--security-opt seccomp=unconfined` to the docker (which we do in `docker-build.sh`).

Reply:
Thanks for figuring this out!